db/new_seeds/scenarios/schools/school.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+module NewSeeds
+  module Scenarios
+    module Schools
+      class School
+        attr_reader :name, :urn, :school, :induction_tutor, :school_cohorts, :partnerships
+
+        def initialize(name: nil, urn: nil)
+          @name = name
+          @urn = urn
+          @school_cohorts = {}
+          @partnerships = {}
+        end
+
+        def build
+          school_attributes = { name:, urn: }.compact
+          @school = FactoryBot.create(:seed_school, **school_attributes)
+
+          self
+        end
+
+        ###
+        # adds a user with an induction coordinator profile for the school
+        ###
+        def with_an_induction_tutor(full_name: nil, email: nil)
```

Comment:
Making this modular is definitely going to be a win, there are so many places this could come in handy, in seeds or tests.